tests/console/yast2_samba.pm

```diff
@@ -230,9 +227,45 @@ sub run {
     send_key 'alt-o';
 
     wait_serial('yast2-samba-server-status-0', 60) || die "'yast2 samba-server' didn't finish";
+}
+
+sub gui_new {
+    assert_screen 'yast2_samba-workgroup';
+    send_key 'alt-n';
+    if (check_screen 'yast2_still_susefirewall2', 10) {
+        record_soft_failure 'bsc#1064405';
+        send_key 'alt-c';
+    }
+    wait_still_screen;
```

Comment:
the default stilltime of 7 seconds is a big high, don't you think?

Reply:
reduce to 3